### DISCUSSION TITLE
[Bug]: Unable to add subscription to my Frigate+ account

### DISCUSSION THREAD

**Discussion by brianfeucht** [blakeblackshear/frigate#13224] [Report a Bug] [answered]
### Describe the problem you are having

When I click the button to add a plus subscription to my Frigate account nothing happens.

### Steps to reproduce

1. Open Frigate+ with an account which does not have a subscription.
2. Click on the Subscription menu item on the left
3. Click on the Upgrade Plan button
4. Click on the Subcribe button

Expected: 
A page where I can send you money

Actual:
Nothing

### Version

N/A

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
instrument.js:109  There are no focusable elements inside the <FocusTrap />
(anonymous) @ instrument.js:109
(anonymous) @ use-focus-trap.esm.js:78
Fi @ react-dom.production.min.js:262
t.unstable_runWithPriority @ scheduler.production.min.js:18
$l @ react-dom.production.min.js:122
Oi @ react-dom.production.min.js:261
(anonymous) @ react-dom.production.min.js:261
I @ scheduler.production.min.js:16
w.port1.onmessage @ scheduler.production.min.js:12
```

### Operating system

Windows

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

N/A

### Any other information that may be helpful

I'm using Edge

**Comment by NickM-27** [comment, marked as answer]
this repo is for frigate itself, you should email blake@frigate.video for things pertaining to frigate+
